### PR TITLE
Added pre-0.0.21 color preset

### PR DESCRIPTION
**Pull request by applejag**
# Description

One of the first issues on this repo was that the color scheme changed (#4). 

## Preview

image

The coloring is not identical in some areas such as `kubectl describe`, but it's close enough. Can't get 100% old scheme as a lot of parsing is done differently now.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## What you changed

- Added color schemes from the pre-0.0.21/hidetatz version

## Why you think we should change it

For people coming from hidetatz's kubecolor then this is a big change. This PR adds back that color scheme, in case they really liked it.

## Related issue (if exists)

Relates to #4

### CODE REVIEW

File: config/preset.go [hunk 1]
```diff
@@ -29,6 +32,10 @@ func (p Preset) String() string {
 		return "dark"
 	case PresetLight:
 		return "light"
+	case PresetPre0021Dark:
+		return "pre-0.0.21-dark"
+	case PresetPre0021Light:
+		return "pre-0.0.21-light"
```
Comment (applejag) [inline config/preset.go:38]: I'm not :100: about the naming here. I would go for something like `legacy-dark`, but I think we should also have a pre-0.3.0 color theme for those that really want it, as I think we should change the default color theme.

Wdyt? @prune998

Comment (prune998) [reply config/preset.go:38]: I think this is fine... let's keep the `pre-xxx` as we can support for future changes too. As long as it is documented I'm fine with it.

**Comment by prune998** [comment]
I'm good with the naming so you can merge if you're good too.

**Comment by applejag** [comment]
> I'm good with the naming so you can merge if you're good too.

Alright. But just had another thought. Do we want to name this `legacy-0.0.20` instead? Or just `v0.0.20`?

Is it more important which version the theme changed (pre-0.0.21), or the last version it had the theme (v0.0.20)?

**Comment by prune998** [comment]
Maybe just setting the version makes sens, so `v0.0.20`. it's simpler. But it's not something people can compute themselves, they have to pick one based on the documentation.... so the easiest the better.

**Comment by applejag** [comment]
Alright! 👌 Just to get this right, you mean "let's go with `pre-0.0.21`", ye?

**Comment by prune998** [comment]
I mean you can chose :) I'm ok with both...

**Pull request merged**